paymentsheet/src/test/java/com/stripe/android/view/ActivityScenarioFactory.kt

```diff
@@ -22,6 +22,7 @@ internal class ActivityScenarioFactory(
         )
     }
 
+    @Suppress("Deprecated")
```

Comment:
Why do we test this in PaymentSheet?

Reply:
Good question! We aren't actually testing the activity itself, we are just using it to test out our functions that take in an activity as a parameter, e.g. the FlowControllerFactory constructor. We should be using an example activity for that, rather than using the BI activities. I will follow up and update those tests

Reply:
Thanks!

Reply:
https://github.com/stripe/stripe-android/pull/9453